packages/plugin-core/src/commands/SetupWorkspace.ts

```diff
@@ -184,11 +192,20 @@ export class SetupWorkspaceCommand extends BasicCommand<
 
   async execute(opts: CommandOpts): Promise<DVault[]> {
     const ctx = "SetupWorkspaceCommand extends BaseCommand";
+    // This command can run before the extension is registered, especially during testing
```

Comment:
Maybe this is less of a problem since this is just a feature flag and eventually we're saying this will be the default so we don't need to check settings anymore - but since the setting is in VSCode settings, we don't actually need the extension up in order to read the setting right? Can we just read the VSCode setting rather than our own workspace config file?

Reply:
You're right, it should just look at the VSCode setting. Let me refactor, which will also remove the need for `isRegistered` since VSCode already has the config ready before the extension even initializes.